docs/dcomibmtoolsattachenable.md

```diff
@@ -52,13 +52,13 @@ A useful reference for information about the Java&trade; Attach API can be found
 
 > The Attach API is an extension that provides a mechanism to attach to a Java virtual machine. A tool written in the Java Language, uses this API to attach to a target  virtual machine and load its tool agent into that virtual machine.
 
-For example, to late attach the IBM&reg; Health Center agent to a virtual machine (VM) that is already running.
+A usage example is to late attach the IBM&reg; Health Center agent to a virtual machine (VM) that is already running. If the VM is started with the Attach API enabled, the Health Center agent can attach and monitor the running processes.
 
```

Comment:
> the Health Center agent can attach and monitor the running processes.

To clarify, if the Health Center agent is started at the same time as the Java application, the agent and the application run in the same JVM, in this case, no need to attach to it in this case.

Reply:
Removed the second sentence "If the VM is started with the Attach API enabled, the Health Center agent can attach and monitor the running processes."